WordPress/Classes/ViewRelated/Blog/Blog Dashboard/HostCollectionViewCell.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import SwiftUI
+
+protocol Hostable {
+    associatedtype Content: View
+    var hostController: UIHostingController<Content>? { get }
+    var hostedView: Content? { get set }
+}
+
+class HostCollectionViewCell<Content>: UICollectionViewCell, Hostable where Content: View {
```

Comment:
What do you think about adding a comment stating what this class does?

Reply:
Fixed! f3ec478